PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java

```diff
@@ -106,6 +106,7 @@
         AzureEventBus.after("mysql|server.start", this::onMySqlServerStatusChanged);
         AzureEventBus.after("mysql|server.restart", this::onMySqlServerStatusChanged);
         AzureEventBus.after("mysql|server.stop", this::onMySqlServerStatusChanged);
+        AzureEventBus.after("mysql|server.delete", this::onMySqlServerStatusDeleted);
```

Comment:
![Uploading image.png…]()

Reply:
I think it's not a good idea. `onMySqlServerStatusDeleted` keeps the same style with `onMySqlServerStatusDeleting`, just like `onMySqlServerStatusChanging` with `onMySqlServerStatusChanged`

Reply:
already handled this case

Reply:
please note `delete` is different with other actions.

Reply:
even so, you need to delete line 100 to 105

Reply:
deleted.